### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by srishtigautam21**
Added loginPage and requireAuth
Files to check:
LoginPage.jsx
LoginPage.css
RequireAuth.js
AuthContext.js
App.js
Navbar.jsx

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
Nice code looks good to merge

File: src/component/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -13,9 +14,25 @@ const Navbar = () => {
...
         </div>
         <li className='icons-alignment'>
-          <Link className='link ecom-link-color' to='/homePage'>
-            <UserIcon className='nav-icons' />
-          </Link>
+          <div className='dropdown'>
+            <Link
+              className='link ecom-link-color'
+              to='/homePage'
+              onClick={() => setOpen(!open)}
```
Comment (iRohitGaur) [inline src/component/Navbar/Navbar.jsx]: suggestion: do a functional update here

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Good to merge 👍🏻

File: src/component/Navbar/Navbar.jsx (hunk 1 above)
Comment (srishtigautam21) [reply src/component/Navbar/Navbar.jsx]: done

**Pull request merged**